docs/src/routes/docs/_layout.svelte

```diff
@@ -149,6 +155,10 @@
   <DesktopNavigation {places} segment={segment || ''} />
   <article>
     <slot />
+    <Button on:click={scrollToTop}>
```

Comment:
I think it would look better `filled` and centered.
Also, how about hiding it when the page is too short to have a scrollbar (such as the [docs homepage](https://illright.github.io/attractions/202/docs)? This can be done with something like:
```js
let scrollbarVisible;
onMount(() => {
    scrollbarVisible = window.innerHeight < document.body.scrollHeight;
});
```

Reply:
I know, there are lots of things we could do to make it better, but the point is, I want to rethink the design of the docs entirely and then implement all sorts of nice stuff. For now I just threw it in just to be there

Reply:
I get it, but honestly I think that they are great as they currently are; don't overthink it. In any way, I don't think it's much work adding a prop and centering it (and even the 2 extra lines of code)